Explorer/Assets/DCL/AvatarRendering/Wearables/Helpers/WearableComponentsUtils.cs

```diff
@@ -63,11 +77,41 @@ public static GetWearablesByPointersIntention CreateGetWearablesByPointersIntent
                 return null;
             }
 
+            if (attachment.ManifestResult?.Asset == null)
+            {
+                try
+                {
+                    attachment.ManifestResult  = new StreamableLoadingResult<SceneAssetBundleManifest>(
+                        await LoadWearableAssetBundleManifestUtils.LoadWearableAssetBundleManifestAsync(assetBundleURL, attachment.GetHash(), ReportCategory.WEARABLE, cancellationTokenSource?.Token ?? new CancellationToken()));
+                }
+                catch (Exception e)
+                {
+                    ReportHub.Log(ReportCategory.WEARABLE, $"Wearable {attachment.GetHash()} doesnt have a manifest");
```

Comment:
Perhaps you could also add in the message: `$"Cannot load the thumbnail of the wearable {attachment.GetHash()} doesnt have an AB manifest. Trying to get the texture through http.."`

Reply:
Done!

Reply:
The messaged you add there is way more descriptive. Copying it